### PR TITLE
Enh/#12 billing without postal

### LINKED ISSUE #12: Handle billing for insurance companies without postal address
## Problem

The billing validator (XRechnung, not the diga specific part, but CII part) does not accept bills without the buyer's postal address. However, there is at least one (BH, BKK Herkules) and probably more companies in the mapping file (Krankenkasse....DiGA.xml) which do not have postal information. How do we handle those?

## Solution

?

### PR DESCRIPTION
**Pull request by fongie**
# Pull Request

### Description

as mentioned in #12, if the postal info is missing we can just put dummy values in the invoice

### Closes

closes #12

### CODE REVIEW

File: src/main/java/com/alextherapeutics/diga/DigaApiClient.java [hunk 1]
```diff
@@ -160,9 +160,21 @@ public DigaInvoiceResponse sendTestInvoiceRequest(DigaInvoice invoice, String in
                 .insuranceCompanyIKNumber(healthInsuranceInformation.getIKAbrechnungsstelle())
                 .buyerCompanyCreditorIk(healthInsuranceInformation.getIKDesRechnungsempfaengers())
                 .insuranceCompanyName(healthInsuranceInformation.getNameDesKostentraegers())
-                .buyerCompanyPostalCode(healthInsuranceInformation.getPLZ())
-                .buyerCompanyAddressLine(healthInsuranceInformation.getStrassePostfach() + " " + healthInsuranceInformation.getHausnummerPostfachnummer())
-                .buyerCompanyCity(healthInsuranceInformation.getOrt())
+                .buyerCompanyPostalCode(
+                        healthInsuranceInformation.getPLZ() != null
+                                ? healthInsuranceInformation.getPLZ()
+                                : DigaBillingInformation.INFORMATION_MISSIONG
```
Comment (StaffanKilnes) [inline src/main/java/com/alextherapeutics/diga/DigaApiClient.java:166]: Found an extra O! Should be INFORMATION_MISSING

**Review by StaffanKilnes — CHANGES_REQUESTED** [review]
Found a spelling error, otherwise nice!

**Comment by fongie** [comment]
woops hehe, fixed

**Review by StaffanKilnes — APPROVED** [review]
Awesome!

**Pull request merged**